recipe/meta.yaml

```
@@ -132,13 +140,13 @@ outputs:
       commands:
         - Rscript test-r-xgboost.r
 
-  - name: r-xgboost-cpu
-    requirements:
-      host:
-        - r-base
-      run:
-        - r-base
-        - {{ pin_subpackage('r-xgboost', exact=True) }}
+  # - name: r-xgboost-cpu
+  #   requirements:
+  #     host:
+  #       - r-base
+  #     run:
+  #       - r-base
+  #       - {{ pin_subpackage('r-xgboost', exact=True) }}
```

As this is an empty package, you should be able to uncomment it.
```suggestion
  - name: r-xgboost-cpu
    requirements:
      host:
        - r-base
      run:
        - r-base
        - {{ pin_subpackage('r-xgboost', exact=True) }}
```